test/integration/sign-out-not-signed-in-test.js

```diff
@@ -0,0 +1,28 @@
+var store = require('humble-localstorage')
+var test = require('tape')
+
+var Account = require('../../index')
+
+var baseURL = 'http://localhost:3000'
+
+test('sign out without being signed in', function (t) {
+  t.plan(2)
+
+  // simulate user who has not yet signed in
+  store.clear()
+  store.setObject('account', {})
```

Comment:
When simulating a user who is not signed in, Is it sufficient to set the account object to an empty object literal, or should there be more items in the object?  From the code, it seems that having a session object inside of the account object implies that the user is logged in, so I wouldn't want to add a session object, but I didn't know if any other information is necessary in the account object to simulate a user who is not signed in.